src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
@whipermr5 Refer to my https://github.com/TEAMMATES/teammates/pull/7774#discussion_r127772364 regarding the wrapper approach. Basically I need help figuring out how I can create a wrapper around the pseudo :before element.